pynest/nest/tests/test_aeif_lsodar.py

```diff
@@ -249,10 +249,10 @@ def assert_pass_tolerance(self, rel_diff, di_tol):
                                     model, var, diff, di_tol[model][var]))
 
     @unittest.skipIf(not HAVE_GSL, 'GSL is not available')
-    def test_closeness_nest_lsodar(self):
-        '''
-        Compare models to the LSODAR implementation.
-        '''
+    def testClosenessNestLSODAR(self):
+
+        # Compare models to the LSODAR implementation.
```

Comment:
@gtrensch Tests should be commented using docstrings and the first line should give a concise description of the test. We haven't been particularly systematic about this, but changing docstrings to comments, is in my opinion the wrong way to go. In this particular case, the text is also fine as a docstring.